subgraph-kovan-staging.yaml

```diff
@@ -132,3 +134,85 @@ dataSources:
         - function: update(bytes32,uint256)
           handler: handleNftFeedUpdate
       file: ./src/mapping.ts
+
+
+  # Kovan V3
+  - kind: ethereum/contract
+    name: SeniorTranche2
+    network: kovan
+    source:
+      address: "0x0f4ee0d02c98bb4443ff88ffafd76fc8ad3d82f4"
+      abi: SeniorTranche
+      startBlock: 20804424
+    mapping:
+      kind: ethereum/events
+      apiVersion: 0.0.3
+      language: wasm/assemblyscript
+      entities:
+        - Pool
+      abis:
+        - name: SeniorTranche
+          file: ./abis/SeniorTranche.json
+      callHandlers:
+        - function: file(bytes32,uint256)
+          handler: handleSeniorTrancheFile
```

Comment:
This should not be needed anymore – that call only exists for v2, right?

Reply:
Instead, we should call `handleAssessorFile` if the assessor's `file` should be called.